src/features/list-view/styles/ListView.scss

```diff
@@ -51,4 +51,12 @@
     .ReactVirtualized__Grid {
         outline: none;
     }
+
+    .ReactVirtualized__Grid__innerScrollContainer {
```

Comment:
Is there any way we can pass a class of some kind into `ReactVirtualized` so that we can style that instead of this class directly? It's usually undesirable to style a class we don't own.

Reply:
Discussed with Sam, will be adding a TODO regarding the limitations of our current approach and plans for a more ideal solution.